sdk-rs/src/lib.rs

```diff
@@ -264,14 +264,32 @@ impl AccountProvider for WsAccountProvider {
 #[must_use]
 pub struct DriftClient<T: AccountProvider> {
     backend: &'static DriftClientBackend<T>,
+    wallet: Wallet,
+    pub active_sub_account_id: u8,
+    pub sub_account_ids: Vec<u8>,
+    compute_unit_params: ComputeUnitParams
 }
 
 impl<T: AccountProvider> DriftClient<T> {
-    pub async fn new(context: Context, account_provider: T) -> SdkResult<Self> {
+    pub async fn new(
+        context: Context, 
+        account_provider: T, 
+        keypair: Keypair, 
+        active_sub_account_id: Option<u8>, 
+        sub_account_ids: Option<Vec<u8>>,
+        compute_unit_params: Option<ComputeUnitParams>
```

Comment:
use a struct for the optional stuff, will make a cleaner api e.g

```suggestion
   // it just works constructor (uses ClientOpts::default() )
   pub async fn new(
        context: Context, 
        account_provider: T, 
        keypair: Keypair, 
   )
   
   // more advanced constructor
   pub async fn with_opts(
        context: Context, 
        account_provider: T, 
        keypair: Keypair, 
        client_opts: ClientOpts,
   )
```